### PR TITLE
Feat/section tag

### PR DESCRIPTION
**Pull request by Alez05**

### CODE REVIEW

**Review by flivanov92 — APPROVED** [review]
👍

**Review by antonia2203 — COMMENTED** [review]
💯

File: src/alez05/article-tag.pug [hunk 1]
```diff
@@ -0,0 +1,25 @@
+html
+  head
+    title Dressarossa
+  body
+    div
+      h1 The article element
+    div
+      article
```
Comment (MariaTeo) [inline src/alez05/article-tag.pug:8]: this tag is empty.

**Review by MariaTeo — CHANGES_REQUESTED** [review]
you also committed more files than section-tag.pug. at least add some content in the empty tag

File: src/alez05/section-tag.pug [hunk 2]
```diff
@@ -0,0 +1,20 @@
+html
+  head
+    title Whole Cake Island
+  body
+    h1 The section element
+    p This is it
+    section
+    h2 WWF History
+    p The World Wide Fund for Nature (WWF) is an international organization working on issues regarding the conservation, research and restoration of the environment, formerly named the World Wildlife Fund. WWF was founded in 1961.
```
Comment (marianzburlea) [inline src/alez05/section-tag.pug]: 8 and 9 should be children to **section**, that's why we use the tag, to wrap other tags

File: src/alez05/section-tag.pug [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    section
+    h2 WWF's Symbol
```
Comment (marianzburlea) [inline src/alez05/section-tag.pug]: 11 to 20 should be inside the **section** tag

File: src/alez05/article-tag.pug [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    div
+      h2 Crunchyroll
+      p Crunchyroll is the website where you can find loads of anime, as well as games. You can find merchandise with cups and t-shirts as well.
+    div
+      article
+      h2 Edward Newgate
```
Comment (marianzburlea) [inline src/alez05/article-tag.pug]: h2 and p, inside **article**

File: src/alez05/iframe.pug [hunk 5]
```diff
@@ -0,0 +1,22 @@
+html
+  head
+    title Iframe
+  body
+    h1 Today we will learn about Iframe
+    p What is Iframe?
+    iframe(
+      src="https://6portals.com" title="Buy the best merch in here"
```
Comment (marianzburlea) [inline src/alez05/iframe.pug]: src and title on separate lines
```
src="asdas"
title="aasdasd"
```

**Review by marianzburlea — CHANGES_REQUESTED** [review]

**Review by Vivy — APPROVED** [review]
nice 1

**Review by MariaTeo — APPROVED** [review]

**Review by marianzburlea — APPROVED** [review]
💯

**Pull request merged**